wiki/osu!dev_Discord_server/zh.md

```diff
@@ -0,0 +1,37 @@
+---
+tags:
+  - development
+  - server
+  - 开发
+---
+
+# osu!dev Discord 服务器
+
+*警告：中国大陆无法访问 Discord，后果请自行承担。*
+
+**osu!dev** 是官方 [Discord](https://discordapp.com "Discord's website") 服务器，它用于 osu! 的开源和各种社区项目。这有助于促进 osu! 的开发并与 [osu! 开发团队](/wiki/People/The_Team) 进行联系的主要场所。
```

Comment:
the text inside the link ("Discord's website") should be translated, it appears when you hover over the link

also happens later with the "on GitHub" links